src/task.js

```diff
@@ -0,0 +1,123 @@
+const { taskForm } = document.forms;
+const { description } = taskForm;
+
+const mainTasks = document.querySelector('.main-tasks');
+
+export default class Task {
+  static showTasks = () => {
+    this.tasks = [];
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) !== null ? (this.tasks = JSON.parse(localStorage.getItem('tasks'))) : [];
+    mainTasks.innerHTML = '';
+    this.tasks.forEach((task) => {
+      let taskTemplate = `<div class="tasks-container" id="${task.index}">`;
+      if (task.completed) {
+        taskTemplate += `
+                  <span class="material-icons check" id="${task.index}">
+                    done
+                  </span>
+                  <input type="text" class="description completed" id="${task.index}" value="${task.description}">
+                  <span class="edit-icon material-icons">delete<span>
+                  </div>
+                  <hr>
+                  `;
+      } else {
+        taskTemplate += `
+                  <span class="material-icons check" id="${task.index}">
+                    check_box_outline_blank
+                  </span>
+                  <input type="text" class="description" id="${task.index}" value="${task.description}">
+                  <span class="edit-icon material-icons">delete<span>
+                  </div>
+                  <hr>
+                  `;
```

Comment:
Please adhere to the DRY principle by avoiding code repetition. The code snippet above shows that you are only changing the message for your `span` with class `check` blocks in both ` if/else` conditions. 
- To optimize your code, place the code that needs to render based on the condition inside the `if/else` block and the remaining code outside of it. This will help eliminate duplication and make your code more maintainable.